.gitignore

```diff
@@ -6,3 +6,4 @@ my-app*
 template/src/__tests__/__snapshots__/
 lerna-debug.log
 npm-debug.log
+.idea/
```

Comment:
Add .idea to your global .gitignore not this one. See https://help.github.com/articles/ignoring-files/ for details